static/manifest.json

```diff
@@ -0,0 +1,84 @@
+{
+  "name": "brid.gy",
+  "short_name": "brid.gy",
+  "description": "📣 Connects your web site to social media. Likes, retweets, mentions, cross-posting, and more...",
+  "id": "/",
+  "start_url": "/",
+  "scope": "/",
+  "lang": "en",
+  "dir": "ltr",
+  "theme_color": "#428ef4",
+  "background_color": "#ffffff",
+  "display": "standalone",
+  "orientation": "portrait-primary",
+  "related_applications": [],
+  "prefer_related_applications": false,
+  "icons": [
+    { "src": "manifest-icons/icon-72x72.png", "sizes": "72x72", "type": "image/png" },
```

Comment:
These aren't all necessary, right? I'd rather avoid maintaining lots of new logo files. Should we maybe just use some of the existing `static/bridgy_logo*` files?

Reply:
Many different devices types needs different icons. These are the minimum for Chrome, Safari, Edge and so on.

Reply:
Huh. Do those browsers not resize existing icons if there's not already a preferred size? Got a link with details?

Reply:
It's best practice many of these generators also use and low level for the main OS's Android and iOS. For one of my PWAs, I took the effort to look at all the specs and came up with over 80 icons, including the maskable stuff :)

https://stackoverflow.com/questions/48839338/which-icon-sizes-are-required-for-progressive-web-apps-pwa-as-of-q1-2018

Reply:
Hmm. It sounds like the major browser and platforms generally scale down icons fine: https://github.com/GoogleChrome/lighthouse/issues/291#issuecomment-219074630

Reply:
My experience is rather "should", but it is actually more of a marginal problem. Therefore, two or three slightly larger icons are also adequate

Reply:
Great! There are currently six JPGs with opaque white background and five PNGs with transparent background, across a range of sizes and aspect ratios. Maybe we start with these, either some or all of them, and see how it goes?

* bridgy_logo.jpg
* bridgy_logo_128.jpg
* bridgy_logo_16.jpg
* bridgy_logo_75.jpg
* bridgy_logo_square.jpg
* bridgy_logo_square_1024.jpg
* bridgy_logo_with_alpha.png
* bridgy_logo_with_alpha_128.png
* bridgy_logo_with_alpha_16.png
* bridgy_logo_with_alpha_75.png
* bridgy_logo_with_alpha_square_1024.png